ethportal-peertest/README.md

```diff
@@ -0,0 +1,10 @@
+# ethportal-peertest
+
+Run a portal network node that you want to test and pass node's Enr as a target node argument.
```

Comment:
I would like to see some expansion / more detail on how to use this. But, I totally think that will come naturally as the test harness develops.